packages/workbench/src/app/modules/login/login.component.ts

```diff
@@ -0,0 +1,101 @@
+import {Component, CUSTOM_ELEMENTS_SCHEMA, OnInit} from '@angular/core';
+import {FormBuilder, FormGroup, ReactiveFormsModule, Validators} from "@angular/forms";
+import {ActivatedRoute, Router} from "@angular/router";
+import {TranslocoPipe, TranslocoService} from "@jsverse/transloco";
+import {
+  OntoToastrService,
+  ServiceProvider,
+  SecurityService,
+  UrlPathParams,
+  AuthenticationService
+} from '@ontotext/workbench-api';
+import {CommonModule} from "@angular/common";
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [
+    TranslocoPipe,
+    ReactiveFormsModule,
+    CommonModule
+  ],
+  schemas: [CUSTOM_ELEMENTS_SCHEMA],
+  templateUrl: './login.component.html',
+  styleUrl: './login.component.scss'
+})
+export class LoginComponent implements OnInit {
+  private readonly toastrService = ServiceProvider.get(OntoToastrService);
+  private readonly securityService =  ServiceProvider.get(SecurityService);
+  private readonly authenticationService =  ServiceProvider.get(AuthenticationService);
+
+  loginForm!: FormGroup;
```

Comment:
I don't really like the using the `definite assignment assertion operator` `!` if it is not required.

Both of these `loginForm` and `returnUrl` can be assigned in the constructor.

Reply:
fixed